extensions/windows-event-log/wel/MetadataWalker.cpp

```diff
@@ -0,0 +1,116 @@
+#include "MetadataWalker.h"
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace wel {
+
+bool MetadataWalker::for_each(pugi::xml_node &node) {
```

Comment:
This seems to be const, too.

Reply:
The node itself is, I agree, but I didn't see modification on the MetadataWalker itself.